### PR TITLE
checking Accessibility of Portfolio Page

### PR DESCRIPTION
**Pull request by bhushan354**
- Page Title : Changed title from "portfolio-website" to "Bhushan's-Portfolio".
- Text Headlines : No issue found.
- Image Alternatives : Added proper alternative texts.
- Interaction : No issue found.
- Moving Content : No issue found.
- Resizing : No issue found.
- Color contrast : Fixed the contrast and added active, disabled & hover .
- Multimedia : No issue found.
- Basic structure of the page : Fixed Horizontal scrolling .

Thank you ,
Bhushan.

### CODE REVIEW

**Review by wayungi — CHANGES_REQUESTED** [review]
Hi <STUDENT_MENTION>, 

image


**Congratulations on the good job done so far!** 🤝 

There are some issues that you still need to work on to go to the next project but you are almost there! 😄 

## Highlights ⚡ ⚡ 
- [ ] Good commit messages ✔️ 
- [ ]  Correct GitHub flow ✔️ 
- [ ] Descriptive PR body ✔️ 

## Required Changes ♻️
_Check the comments under the review._
- [ ] Please make sure your headings are in a hierachy. Alos note that a page can only have one main heading `h1`. 
titile

- [ ] Your page has also failed the resize test. Your texts over lap when scale to 200%.
resize




## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by bhushan354** [comment]
Hello sir , Thank you for giving your valuable review I have made changes as you suggested.

- Used only one h1 tag and also made all headings in hierarchy.

- Also fixed the resizing issue which was causing overlapping over text :
Screenshot (54)

Thank You,
Bhushan.

File: index.html
Comment (KanzaTahreem) [inline index.html:1]: - [ ] As suggested previously, please make sure that you have only one `h1` per page and that should be the main headline. You can use `h2, h3, ...` for other heading but make sure that they are hierarchical such as mentioned below 👇 

```
Heading Level 1 <h1>
  Heading Level 2 <h2>
    Heading Level 3 <h3>
    Heading Level 3 <h3>
  Heading Level 2 <h2>
    Heading Level 3 <h3>
      Heading Level 4 <h4>
      Heading Level 4 <h4>
  Heading Level 2 <h2>
```
image

**Review by KanzaTahreem — CHANGES_REQUESTED** [review]
# Changes Required 🔁

Hi @bhushan354 🙋‍♀️

Good job so far! I really appreciate the effort you put in!  **_However, there are few this that still need to be fixed. But you are almost there!_** 💪 🏁

so close


## Highlights 🎯

There are some excellent points that you have made perfectly. Keep up the great work! 👏

- ✅ Descriptive PR and README file
- ✅ Evaluated website's accessibility.
- ✅ Correct GitHub Flow

## Required Changes 🛑

_Please check the comments under the review._

## Optional suggestions ❕

N/A

Cheers and Happy coding!👏👨‍💻💻

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by bhushan354** [comment]
Hello Kanza, Thank you for reviewing my code  I have made changes according to your review.

- Used only one h1 tag this time and used h2 tag for level 2 headings.

image

Thank you ,
Bhushan

**Review by Shedrack-Sunday — CHANGES_REQUESTED** [review]
Hi @bhushan354       👋

Congratulations! 🎉 Your project is complete and ready to be merged! 🚀

image


## Highlights
- Good job with the use of GitHub features✔️ 
- Kudos on implementing the changes as requested ✔️
 

## Optional Suggestions 💡
While these comments with the [OPTIONAL] prefix won't prevent the approval of this pull request, I strongly recommend taking them into account as they can improve your code. Some of them were missed by the previous reviewer and addressing them will really enhance your application.
Cheers, and Happy coding!👏👏👏

If you have any questions or need clarification on anything, feel free to leave a comment in the pull request thread, and don't forget to tag me so I'll receive the notification.

As per the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561), you have a limited number of reviews per project. If you feel that the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**